atari_py/package_data.txt

```diff
@@ -367,69 +368,7 @@ ale_interface/src/os_dependent/SettingsUNIX.hxx
 ale_interface/src/os_dependent/SettingsWin32.cxx
 ale_interface/src/os_dependent/SettingsWin32.hxx
 ale_interface/src/os_dependent/module.mk
+atari_roms/tetris.bin
```

Comment:
is that ok if we keep tetris? I understand we want it for tests, but would not get us in just as much of a legal pickle as the rest of the roms?

Reply:
This is a non-commercial rom as mentioned here: https://github.com/openai/atari-py/pull/62/files#diff-04c6e90faac2675aa89e2176d2eec7d8R37